R/github-actions.R

```diff
@@ -163,6 +165,9 @@ use_github_action <- function(name,
   if (open && new) {
     edit_file(proj_path(save_as))
   }
+  if (!is.null(readme)) {
+    ui_todo("Learn more at <{readme}>")
```

Comment:
I feel like we might have some other convention going with URLs but I can't search around for that atm.

Reply:
I looked for `ui_url()` but didn't see anything. We can do a thorough pass if/when the IDE gets support for turning these into "real" links.